### DISCUSSION TITLE
RSS Feed - Dumb question...

### DISCUSSION THREAD

**Discussion by Andy-ABTec** [mmistakes/minimal-mistakes#2988] [Q&A] [answered]
Apologies, I've been poking around in the docs and not found the solution...

How do I remove the RSS Feed link from the footer?

Any help/advice gratefully accepted...

**Comment by mmistakes** [comment, marked as answer]
You'll have to pull out this line from `_includes/footer.html`

```html
<li><a href="{% if site.atom_feed.path %}{{ site.atom_feed.path }}{% else %}{{ '/feed.xml' | relative_url }}{% endif %}"><i class="fas fa-fw fa-rss-square" aria-hidden="true"></i> {{ site.data.ui-text[site.locale].feed_label | default: "Feed" }}</a></li>
```

https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/footer.html#L21

**Comment by Andy-ABTec** [comment]
Star! 🥇 Many thanks, works like a charm...

**Comment by hasezoey** [comment]
could this be made an config option?

**Comment by stefanbschneider** [comment]
How would this work if I use the remote theme?

**Comment by mmistakes** [comment]
@stefanbschneider It works the same way as gem based theme. Copy `_includes/footer.html` to your local repo, make whatever changes you need.

Local copies of `_includes`, `_layouts`, etc. override those in the theme. There's more documentation on this at the [official Jekyll docs](https://jekyllrb.com/docs/themes/#understanding-gem-based-themes).

**Comment by sstringer00100** [comment]
The fix works like a charm, but now disables my email link in the both the footer and sidebar.

**Comment by twobiers** [comment]
For anyone coming from Google, the feed can be disabled by setting the follwing configuration property in `_config.yml`:
```yml
atom_feed:
  hide: true
```

**Reply by pmatulis** [reply]
I'm not sure what the last commentator meant by *anyone coming from Google* but I used the suggested config and the link was removed.

**Reply by mmistakes** [reply]
I think what OP meant by “Anyone coming from Google” is *anyone who searched on Google for how to disable the RSS feed link in Minimal Mistakes here’s the magic spell to add to your _config.yml to make that happen.*
